### ISSUE TITLE
autocompletion of \cite{ breaks with labeled bibliography

### ISSUE THREAD

**Issue by SigridK** [SublimeText/LaTeXTools#717]
When using `\addbibresource[label=primary]{bib1.bib}` autocompletion breaks if the label is referred like this: `\begin{refsection}[primary]` but not if the .bib-file is referred like this: `\begin{refsection}[bib1.bib]`

Error message reads: `An error occurred while trying to run kpsewhich. TEXMF tree could not be accessed.`

**Comment by ig0774** [comment]
We currently don't have support for bibresource labels, which is (in a way) why you are getting the error.

The error is raised because LaTeXTools finds the string "primary" and doesn't finding a corresponding bib file, so it tries to to search for a file called "primary.bib" using kpsewhich, a standard tool included with LaTeX distributions that searches the various folders of the TeX distribution. Since it doesn't find the file "primary.bib", it displays the error. The error message is intended for cases where someone entered a bibliography file that should be somewhere in the various paths TeX will search but the file could not be found.

Long story short, that is, right now, the expected behaviour, though that error message could probably be more clear.
